rpc/rpc.h

```diff
@@ -161,16 +161,25 @@ namespace rpc
         virtual int set_close_notify(Notifier notifier) = 0;
 
         // can be invoked concurrently by multiple threads
-        // if have the ownership of stream, `serve` will delete it before exit
-        virtual int serve(IStream* stream, bool ownership_stream = false) = 0;
+        virtual int serve(IStream* stream) = 0;
+
+        [[deprecated]]
```

Comment:
use ```__attribute__((deprecated))``` to fit for C++11 (and gcc 4.9)

similar for ```new_skeleton()```

Reply:
It's contradictory to support Clang/Windows and old gcc compiler at the same time